webcompat/static/js/lib/wizard/steps/submit.js

```diff
@@ -53,15 +53,14 @@ const submitForm = function () {
   const dfd = $.Deferred();
   const formEl = form.get(0);
 
-  sendAnalyticsEvent("success", "end");
-
   formEl.submit();
   dfd.resolve();
   return dfd.promise();
 };
 
 const onFormSubmit = (event) => {
   event.preventDefault();
+  sendAnalyticsEvent("success", "end");
```

Comment:
I've moved analytics success event to happen a bit earlier, since I've noticed that it's not being tracked due to the url change after form submit. 
(this is not related to the changes addressed in this PR, but it's probably not worth creating a separate issue)